pages/platform/ai/apps_01_tokens/guide.en-gb.md

```diff
@@ -0,0 +1,151 @@
+
+![app list](images/03-app-list.png)
+
+Once on the token management tab, simply click on `New Token`{.action}.
+
+![token list new](images/04-token-list-new-2.png){.thumbnail}
+
+#### Read token
+
+There are two types of roles that can be assigned to a token:
+
+- AI Platform - Read-only
+- AI Platform - Operator
+
+A Read-only token will only grant you the right to query the deployed AI App while an Operator token would also allow you to manage the AI App itself.
+
+Let us create a token for the AI Apps matching the label `group=A` with read-only access in the GRA cluster.
+To create an AI App token we need to specify 3 parameters:
+
+- The token scope specified through label selectors, a token will be scoped over any AI app matching the set of label selectors. In this case `group=A`
+- The token role: AI Training - Read-only
+- The region (cluster in which are deployed the AI Apps): GRA.
+
+Fill out the form:
+
+![token generation input read](images/05-token-gen-input-read.png){.thumbnail}
+
+Click `Generate`{.action}. Upon success, you are redirected to the token list with the new generated token displayed at the top:
+
+![token generation result](images/06-token-gen-result-read.png){.thumbnail}
+
+Save the token string for later use.
+
+> [!warning] 
+> The token is only displayed once, make sure to save it before leaving the page or you will need to regenerate the token.
+
+This newly generated token provides read access over all resources tagged with the label `group=A` including the ones submitted after the creation of the token.
+
+#### Operator token
+
+An operator token grants read access along with management access for the matching AI apps. This means that you can manage the AI App lifecycle (start/stop/delete) using either the CLI (more info [here](https://docs.ovh.com/gb/en/ai-training/install-client/)) or the [AI Training API](https://gra.training.ai.cloud.ovh.net/) by providing this token.
+
+This Operator token will be scoped on a specific AI App and we will use the default `ovh/id` label to do so (since it is reserved, there is only one AI App that can match this label selector).
+
+- Token scope: `ovh/id=4c4f6253-a059-424a-92da-5e06a12ddffd`
+- The token role: AI Training - Operator
+- The region: GRA.
+
+![token generation input operator](images/07-token-gen-input-op.png){.thumbnail}
+
+Additional information about the use of a token to manage an AI Training resource can be found [here](https://docs.ovh.com/gb/en/ai-training/app-token-cli/#use-the-app-token).
+
+### Using a token to query an AI App
+
+With the token we generated in the previous step we will now query the AI App. For this demonstration, we deployed a simple Hello World app that always responds `Hello, World!`.
+
+In the details of our AI App above the Tags you can get the access URL of your AI App.
+
+#### Browser
+
+If access the AI App access URL from your browser you reach a Login page:
```

Comment:
@christophe-rannou, I do not understand that sentence. Can you maybe rephrase it please ?

Reply:
proposition: "When accessing the AI App via its URL in your browser, you will reach a Login page:"